lineage/states/stateCommon.py

```diff
@@ -12,16 +12,15 @@ def bern_pdf(x, p):
     probability distribution function.
     """
     # bern_ll = self.bern_p**(tuple_of_obs[0]) * (1.0-self.bern_p)**(1-tuple_of_obs[0])
-
+    
     return (p**x) * ((1.0 - p)**(1 - x))
 
 
-def bernoulli_estimator(bern_obs):
+def bernoulli_estimator(bern_obs, old_params, gammas):
     """
     Add up all the 1s and divide by the total length (finding the average).
     """
-    bern_obs = np.append(bern_obs,1)
-    return (sum(bern_obs)) / (len(bern_obs))
+    return sum(gammas*bern_obs) / sum(gammas)
```

Comment:
example of how the participation variable is used in the Bernoulli parameter estimation